File: src/main/java/com/whis/app/service/TestService.java
package com.whis.app.service;

import com.whis.app.mapper.AdminMapper;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

import java.util.Map;

@Component
public class TestService {

    private static Logger logger = LoggerFactory.getLogger(TestService.class);

    private final AdminMapper adminMapper;

    @Autowired
    public TestService(AdminMapper adminMapper) {
        this.adminMapper = adminMapper;
    }

    public void testDb() {
        Map<String, Object> admin = adminMapper.finById(1L);

        logger.info("admin {}", admin);
    }
}

File: src/main/java/com/whis/app/service/TestService.kt
package com.whis.app.service

import com.whis.app.mapper.AdminMapper
import org.slf4j.LoggerFactory
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Component

@Component
class TestService {

    companion object {
        private val logger = LoggerFactory.getLogger(TestService::class.java)
    }

    @Autowired lateinit var adminMapper: AdminMapper

    fun testDb() {
        val admin = adminMapper.finById(1L)

        logger.info("admin {}", admin)
    }


    fun testUtil() {
    }
}
